examples/react-apollo/cypress/support/e2e.ts

```diff
@@ -22,10 +40,54 @@ Cypress.Commands.add('signUpEmailPassword', (email, password) => {
   cy.contains('Continue with email + password').click()
 })
 
+Cypress.Commands.add('signUpEmailPasswordless', (email) => {
+  cy.visit('/sign-up')
+  cy.contains('Continue with passwordless email').click()
+  cy.findByPlaceholderText('Email Address').type(email)
+  cy.contains('Continue with email').click()
+})
+
+Cypress.Commands.add('signInEmailPassword', (email, password) => {
+  cy.visit('/sign-in')
+  cy.contains('Continue with email + password').click()
+  cy.findByPlaceholderText('Email Address').type(email)
+  cy.findByPlaceholderText('Password').type(password)
+  cy.contains('Sign in').click()
+  cy.saveRefreshToken()
+})
+
+Cypress.Commands.add('signInToken', function (path = '/') {
+  cy.visit(path + '#refreshToken=' + this.refreshToken)
+})
+
+Cypress.Commands.add('signOut', () => {
+  cy.contains('Sign Out').click()
+})
+
 Cypress.Commands.add('confirmEmail', (email) => {
   cy.mhGetMailsByRecipient(email)
     .should('have.length', 1)
     .then(([message]) => {
       cy.visit(message.Content.Headers['X-Link'][0])
+      cy.saveRefreshToken()
     })
 })
+
+Cypress.Commands.add('quickSignUp', (givenEmail) => {
```

Comment:
can we rename this to `signUpAndConfirmEmail`, so that it becomes a bit more descriptive?